2/contrib/openshift/CentOS-Base.repo

```diff
@@ -9,34 +9,30 @@
 # remarked out baseurl= line instead.
 #
 # 
-[base]
-name=CentOS-7 - Base
-mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=os&infra=$infra
-#baseurl=http://mirror.centos.org/centos/7/os/$basearch/
-gpgcheck=1
-gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+[BaseOS]
+name=CentOS-8 - Base
+mirrorlist=http://mirrorlist.centos.org/?release=8-stream&arch=$basearch&repo=BaseOS&infra=$infra
+gpgcheck=0
```

Comment:
we need to find a way to restore the gpgcheck 
we can probably find the key file somewhere.

Reply:
yes